packages/wallet/src/GovernanceApiBase.ts

```diff
@@ -0,0 +1,101 @@
+import { AsyncKeyAgent, util as keyManagementUtil } from './KeyManagement';
+import { CSL, Cardano, util } from '@cardano-sdk/core';
+import { Ed25519KeyPair, KeyPair } from './KeyManagement/types';
+import { GovernanceApi } from './types';
+import blake2b from 'blake2b';
+
+export type BuildDelegationProps = {
+  votingPublicKey: Cardano.Ed25519PublicKey;
+  nonce: number;
+  rewardAccount: Cardano.RewardAccount;
+};
+
+export interface SignDelegationProps {
+  votingPublicKey: string;
+  publicStakeKey: string;
+  rewardAccountKey: string;
+  nonce: number;
+}
+
+export enum VotingLabels {
+  DATA = 61_284,
+  SIG = 61_285
+}
+
+export type GetVotingKeyPair = (derivationPath: number[]) => Promise<KeyPair>;
+
+export class GovernanceApiBase implements GovernanceApi {
```

Comment:
Tests are structured differently here - they're under `test/GovernanceApi`. Let's put this file and associated types under `src/GovernanceApi`